src/index.ts

```diff
@@ -344,6 +377,12 @@ function addNextIdMethods(chunks: Code[], config: Config): void {
   `);
 }
 
+function getScopedName(config: Config, typeName: string) {
+  return !!config.typeDeclarationFilePath
+    ? `${config?.typeDeclarationImportName ?? DEFAULT_IMPORT_NAME}.${typeName}`
```

Comment:
I'm fine to leave this as-is, but personally might try using ts-poet to do this as an import; so you could remove `config.typeDeclarationImportName` and instead have this be:

```
return !!config.typeDeclarationFilePath ? imp(`${typeName}@${config.typeDeclarationFilePath}`) : typeName;
```

And then when ts-poet sees the `Import` type in the output it will auto-add an import, i.e.:

https://github.com/stephenh/ts-poet/blob/master/src/Import-tests.ts#L41

Granted, it's a little fancier than a straight forward `import * as ...` but I kinda like it.